packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import UploadIcon from "@paprika/icon/lib/Upload";
+import { UploaderContext } from "../../Uploader";
+import * as div from "./DropZone.styles";
+
+export default function DropZone() {
+  const I18n = useI18n();
+  const uc = React.useContext(UploaderContext);
```

Comment:
You could destructure here instead of using a somewhat ambiguous `uc` name.

Reply:
`uc` stands for `UploaderContext`! I like being deeper in the code and coming across `uc.whatever` and then I know where it gets assing, so I don't have to look up in the rest of the code to see where it is getting assigned a value.  But i'm not married to it, so i'll destructure that and another thing I see.

Maybe we should be destructuring I18n's, too.
`const { t } = useI18n();`